src/diff.js

```diff
@@ -56,15 +56,15 @@ class MonacoDiffEditor extends React.Component {
 
   editorDidMount(editor) {
     this.props.editorDidMount(editor, monaco);
-    editor.onDidUpdateDiff((event) => {
+    editor.onDidUpdateDiff(() => {
```

Comment:
Can you explain this change? In https://github.com/superRaytin/react-monaco-editor/blob/master/src/editor.js#L62, we still pass event.

Reply:
According to the type(enter <https://unpkg.com/monaco-editor@0.12.0/esm/vs/editor/editor.api.d.ts> and search `onDidUpdateDiff`), the callback function has no `event` parameter.

Reply:
And in the vscode repo:
https://github.com/Microsoft/vscode/blob/292732cd19cd4c6b0e11e779d14be480ff186ca8/src/vs/editor/browser/editorBrowser.ts#L776